minimumCostOfBuyingCandies.js

```diff
@@ -0,0 +1,20 @@
+// This file contains solution to the following problem:
+// https://leetcode.com/problems/minimum-cost-of-buying-candies-with-discount/
+
+const minimumCost = (cost) => {
+  const sortedPrices = cost.sort((a, b) => b - a);
+  const freeCandy = 3;
+  let result = 0;
+  let counter = 0;
+  let temp = 0;
+  while (counter < sortedPrices.length) {
+    temp += 1;
```

Comment:
Let's name `temp` in more meaningful name (to represent what value it actually holds in it).